packages/runtime/container-runtime/src/test/blobManager.spec.ts

```diff
@@ -172,13 +173,24 @@ export class MockRuntime
 		this.ops = [];
 	}
 
-	public async processBlobs(resolve = true) {
+	public async processBlobs(resolve: boolean, canRetry: boolean = false, retryAfterMs?: number) {
 		const blobPs = this.blobPs;
 		this.blobPs = [];
 		if (resolve) {
 			this.processBlobsP.resolve();
 		} else {
-			this.processBlobsP.reject(new Error("fake error"));
+			const err = createGenericNetworkError(
+				"fake driver error",
+				// TODO: https://dev.azure.com/fluidframework/internal/_workitems/edit/5913
+				{ canRetry, retryAfterMs },
+				{
+					// To avoid using the default retry time added when we don't have retryAfterMs
+					// in the error properties.
+					retryAfterSeconds: retryAfterMs,
```

Comment:
this looks wrong. seconds !== ms

Reply:
This is a workaround, it shouldn't be there in the first place and it's only for these tests to avoid the timeout. I open a task to fix it